src/components/proposalActions.jsx

```diff
@@ -518,7 +518,8 @@ const ProposalVote = ({
                   }
                 />
               ) : (
-                quorumNeeded && (
+                quorumNeeded &&
+                !proposal.isMinion && (
```

Comment:
I put in a function in `proposalUtils` that checks whether a proposal is from a minion or just involves a minion. Quorum only applies for minion proposals where early execution is possible.

Reply:
@skuhlmann @plor ah my bad - i read the issue in the opposite way this needed to be implemented, was weirded out when I could not find the helper method you referenced in `proposalUtils` and discovered my forked `develop` branch was on a way older version!

going to just go from the main repo going forward - updated the branch to check that it is a `isMinionProposalType` which should be good I believe